docs/guides/faq.md

```diff
@@ -0,0 +1,139 @@
+# FAQ
+
+## Q: What is video.js?
+video.js is an extendable framework/library around the native video element. It does the following:
+  * Offers a plugin API so that different types of video can be handed to the native
+    video element. IE:  HLS, Flash, or HTML5 video.
+  * Unifies the native video api across browsers (polyfilling support for features
+    if necessary)
+  * Offers an extendable and themable UI
+  * Takes care of accessibility for the user (in-progress)
+  * Has a set of core plugins that offer support for tons of additional video formats.
+    * [videojs-contrib-hls](http://github.com/videojs/videojs-contrib-hls)
+    * [videojs-contrib-dash](http://github.com/videojs/videojs-contrib-hls)
+  * Support for DRM video via a core plugin
+    * [videojs-contrib-eme](https://github.com/videojs/videojs-contrib-eme)
+  * Lots of plugins which offer support for all kinds of features. See the [plugin list on videojs.com](http://videojs.com/plugins)
+
+## Q: How do I install video.js?
+Currently video.js can be installed using bower, npm, serving a release file from
+a github tag, or even using a CDN hosted version. For information on doing any of those
+see the [install guide](http://videojs.com/getting-started/).
+
+## Q: What do video.js version numbers mean?
+video.js follows [semver](http://semver.org/) which means that the API should not change
+out from under a user unless there is a major version increase.
+
+## Q: How can I troubleshoot playback issues?
+See the [troubleshooting guide](/docs/guides/troubleshooting.md) for information on troubleshooting playback issues
+
+## Q: A video does not play in a specific browser. Why?
+See the [troubleshooting guide](/docs/guides/troubleshooting.md) for information on
+why this might be happening. Open an [issue on the video.js repo](https://github.com/videojs/video.js/issues).
+Make sure that you follow the [issue template](/.github/ISSUE_TEMPLATE.md) and the
+[contributing guide](/CONTRIBUTING.md#filing-issues) so that we can better assist you
+with your issue.
+
+## Q: Why does the entire video download before playback? Why does the video load for a long time?
+See the [troubleshooting guide](/docs/guides/troubleshooting.md) for information on
+why this might be happening. Open an [issue on the video.js repo](https://github.com/videojs/video.js/issues).
+Make sure that you follow the [issue template](/.github/ISSUE_TEMPLATE.md) and the
+[contributing guide](/CONTRIBUTING.md#filing-issues) so that we can better assist you
+
+## Q: I see an error thrown that mentions `vdata12345`. What is that?
+See the [troubleshooting guide](/docs/guides/troubleshooting.md) for information on
+why this might be happening. Open an [issue on the video.js repo](https://github.com/videojs/video.js/issues).
+Make sure that you follow the [issue template](/.github/ISSUE_TEMPLATE.md) and the
+[contributing guide](/CONTRIBUTING.md#filing-issues) so that we can better assist you
```

Comment:
do we want to have two places discussing the same thing in detail? or should player-workflow link to contributing as well?